### ISSUE TITLE
Wrong Story Links

### ISSUE THREAD

**Issue by DanielleSherman** [writingatlas/feedback#11]
When I click on the story "An Argument in 1934," the site takes me to the page "Eight Pieces for the Left Hand."

<img width="1426" alt="Screenshot 2023-12-27 at 5 16 29 PM" src="https://github.com/writingatlas/feedback/assets/154077569/7f405d38-04db-499f-88fe-30c3ebac441a">

<img width="1438" alt="Screenshot 2023-12-27 at 5 17 03 PM" src="https://github.com/writingatlas/feedback/assets/154077569/68ca6856-1518-4520-8036-02dba8c1af0b">

**Comment by jenny8lee** [comment]
Yes. Known bug. Any story ending with a number takes you the story in our
databases numbered with that number, instead of the actual story. (This is
my fault). We are trying to think of a fix.
